google-cloud-pubsub/google-cloud-pubsub.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "concurrent-ruby", "~> 1.1"
   gem.add_dependency "google-cloud-core", "~> 1.5"
   gem.add_dependency "google-cloud-pubsub-v1", "~> 0.0"
+  gem.add_dependency "opentelemetry-sdk", "~> 1.0"
```

Comment:
We cannot depend on opentelemetry-sdk. Use opentelemetry-api. At most the tests (development dependency) can depend on sdk.

Reply:
Understood. This was a mistake, only the tests depend on sdk. Thanks for catching.

Reply:
Fixed, thanks!